analyzer/src/funTest/assets/projects/synthetic/spdx/package/libs/curl/package.spdx.yml

```diff
@@ -18,7 +18,7 @@ packages:
      IMAP, SMTP, POP3, RTSP and RTMP. libcurl offers a myriad of powerful features."
   copyrightText: "Copyright (c) 1996 - 2020, Daniel Stenberg, <daniel@haxx.se>, and many
     contributors, see the THANKS file."
-  downloadLocation: "git+github.com:curl/curl.git@53cdc2c963e33bc0cc1a51ad2df79396202e07f8"
+  downloadLocation: "git+github.com/curl/curl.git@53cdc2c963e33bc0cc1a51ad2df79396202e07f8"
```

Comment:
Unintended change? Looking at the original syntax of the `downloadLocation` I indeed something is wrong here, but I'll look at that separately. It should not be addressed as part o this commit.

Reply:
Change is also done here: https://github.com/oss-review-toolkit/ort/pull/3521 in [separate commit ](https://github.com/oss-review-toolkit/ort/pull/3521/commits/2941a743c34e844d17199e1add4f54e2abe156a9). I'll remove it here and rebase after above PR is merged.

Reply:
I removed it. #3521 will now do the changes. 
This PR is not relying on those changes, hence no need to wait and rebase.